exp/window_func_test.go

```diff
@@ -151,31 +151,31 @@ func (swfet *sqlWindowFunctionExpressionTest) TestAllOthers() {
 
 	expIsNull := wf.IsNull()
 	swfet.Equal(expIsNull.LHS(), wf)
-	swfet.Equal(expIsNull.Op(), IsOp)
+	swfet.Equal(expIsNull.Op(), EqOp)
```

Comment:
This should be reverted to work like it did previously. See https://stackoverflow.com/questions/9822154/standard-sql-boolean-operator-is-vs-equals-operator is and equality work differently for NULL, TRUE and FALSE.